Assests/Login Button.css

```diff
@@ -0,0 +1,17 @@
+.LoginBotton{
+    width: 80%;
+    height: 45px;
+    font-size: 20px;
+    margin: 30px;
+    cursor: pointer;
+    display:flex;
+    justify-content: center;
+    border: 0px;
+    color: white;
+    border-radius: 2px;
+    background-color:#7B1FA2;
+    background: linear-gradient(right,#7B1FA2, rgb(103, 4, 233));
+    background: -webkit-linear-gradient(right,#7B1FA2, rgb(103, 4, 233));
```

Comment:
-webkit-linear-gradient og -moz-linear-gradient bruger et andet retningssystem end linear-gradient. "right" på -moz-linear-gradient og -webkit-linear-gradient er "to left" linear-gradient